plone/app/dexterity/behaviors/discussion.py

```diff
@@ -1,39 +1,6 @@
-from plone.app.dexterity import _
-from plone.autoform import directives
-from plone.autoform.interfaces import IFormFieldProvider
-from plone.supermodel import model
-from z3c.form.interfaces import IAddForm
-from z3c.form.interfaces import IEditForm
-from zope import schema
-from zope.interface import provider
-from zope.schema.vocabulary import SimpleTerm
-from zope.schema.vocabulary import SimpleVocabulary
+from zope.deferredimport import deprecated
 
-
-options = SimpleVocabulary(
-    [
-        SimpleTerm(value=True, title=_("Yes")),
-        SimpleTerm(value=False, title=_("No")),
-    ]
+deprecated(
+    "IAllowDiscussion is deprecated here. Import from plone.app.discussion.behaviors instead (will be removed in Plone 7)"
```

Comment:
Do we want to say `deprecated` at all? 🤔 

If I see _deprecated_ somewhere I get that the functionality is being __removed__.

Here rather we want to say that it has been __moved__ the two letters' diff makes a difference 😆 

Something like: `Please adjust your imports, IAllowDiscussion is now in p.a.discussion`